### PR TITLE
Add ABACUS/lcao deepks interface to dpgen init and run. 

### PR DESCRIPTION
**Pull request by Liu-RX**
The tests/data/test_coll_abacus.py test is also changed according to ABACUS 2.2.0 MD output format. Because of this, the test cannot pass now. It could pass when dpdata supports ABACUS 2.2.0 MD format.

### CODE REVIEW

**Comment by amcadmus** [comment]
please update the requirement for `dpdata` to 0.2.6 https://github.com/deepmodeling/dpdata/releases/tag/0.2.6

**Comment by Liu-RX** [comment]
> please update the requirement for `dpdata` to 0.2.6 https://github.com/deepmodeling/dpdata/releases/tag/0.2.6

Thank you for the update! I have added the requirement for dpdata and the checks have passed.

**Comment by wanghan-iapcm** [comment]
Please also update the requirement in the setup.py, thank you

https://github.com/deepmodeling/dpgen/blob/37df1297247d10e77f4a0b24afa74d4692bdcae4/setup.py#L20

**Comment by Liu-RX** [comment]
> Please also update the requirement in the setup.py, thank you
> 
> https://github.com/deepmodeling/dpgen/blob/37df1297247d10e77f4a0b24afa74d4692bdcae4/setup.py#L20

setup.py has already been updated.

**Review by wanghan-iapcm — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -614,6 +614,10 @@ The bold notation of key (such aas **type_map**) means that it's a necessary key
 | *fp_style == cp2k*
 | **user_fp_params** | Dict |  |Parameters for cp2k calculation. find detail in manual.cp2k.org. only the kind section must be set before use.  we assume that you have basic knowledge for cp2k input.
 | **external_input_path** | String |  | Conflict with key:user_fp_params, use the template input provided by user, some rules should be followed, read the following text in detail.
+| *fp_style == ABACUS*
+| **user_fp_params** | Dict |  |Parameters for ABACUS INPUT. find detail [Here](https://github.com/deepmodeling/abacus-develop/blob/develop/doc/input-main.md#out-descriptor). If `deepks_model` is set, the model file should be in the pseudopotential directory.
```
Comment (AnguseZhang) [inline README.md:618]: This link https://github.com/deepmodeling/abacus-develop/blob/develop/doc/input-main.md#out-descriptor has been invalid. Please check.

File: README.md [hunk 2]
```diff
@@ -614,6 +614,10 @@ The bold notation of key (such aas **type_map**) means that it's a necessary key
 | *fp_style == cp2k*
 | **user_fp_params** | Dict |  |Parameters for cp2k calculation. find detail in manual.cp2k.org. only the kind section must be set before use.  we assume that you have basic knowledge for cp2k input.
 | **external_input_path** | String |  | Conflict with key:user_fp_params, use the template input provided by user, some rules should be followed, read the following text in detail.
+| *fp_style == ABACUS*
+| **user_fp_params** | Dict |  |Parameters for ABACUS INPUT. find detail [Here](https://github.com/deepmodeling/abacus-develop/blob/develop/doc/input-main.md#out-descriptor). If `deepks_model` is set, the model file should be in the pseudopotential directory. 
+| **fp_orb_files** | List |  |List of atomic orbital files. The files should be in pseudopotential directory. 
+| **fp_dpks_descriptor** | String |  |DeepKS descriptor file name. The file should be in pseudopotential directory.
```
Comment (AnguseZhang) [inline README.md:620]: DeePKS?

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
+	"train":[
+	{
+	"machine":{
+		"machine_type": "slurm",
+		"host_name": "localhost",
+		"port": 22,
```
Comment (AnguseZhang) [inline examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json:7]: Could you please update examples compatible with the new version of `dpdispatcher` ?   Search "new dpdispatcher update note" in the manual https://github.com/deepmodeling/dpgen#set-up-machine

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/param.json [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+      "learning_rate": {
+        "type":              "exp",
+        "start_lr":          0.001,
+        "decay_steps":       180,
+        "decay_rate":        0.95
+    },
+      "training": {
+        "systems":           [],
+        "set_prefix":        "set",
+        "stop_batch":        3000,
```
Comment (AnguseZhang) [inline examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/param.json:72]: It seems this is DeePMD-kit 1.x version's parameters. Could you please update the example for DeePMD-kit 2.x?  See https://github.com/deepmodeling/dpgen/blob/devel/examples/run/dp2.x-lammps-vasp/param_CH4_deepmd-kit-2.0.1.json

**Review by AnguseZhang — CHANGES_REQUESTED** [review]
A few changes are suggested.

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json (hunk 3 above)
Comment (Liu-RX) [reply examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json:7]: Maybe I should remove the machine.json here and the same file in ABACUS-pw? The machine json file has no difference with the vasp example except for the "fp" command.

Comment (Liu-RX) [reply examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json]: I have updated the file according to the https://github.com/deepmodeling/dpgen#set-up-machine. You could remove the machine.json as well. I think this example file is not that necessary.

File: README.md (hunk 2 above)
Comment (Liu-RX) [reply README.md]: Yes! Thank you for pointing out the problem.

File: README.md (hunk 1 above)
Comment (Liu-RX) [reply README.md]: I have fixed it.

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/param.json (hunk 4 above)
Comment (Liu-RX) [reply examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/param.json]: I have updated the "training" parameters in both this file and the abacus-pw example file.

**Review by AnguseZhang — APPROVED** [review]

**Comment by AnguseZhang** [comment]
The failure of read-the-doc build will be solved in https://github.com/deepmodeling/dpgen/pull/710

**Pull request merged**